coverage_map/Cargo.toml

```diff
@@ -10,7 +10,6 @@ edition.workspace = true
 [dependencies]
 chrono = { workspace = true }
 h3o = { workspace = true }
```

Comment:
`h30`, `helium-proto`, and `uuid` are also candidates for removal.

Reply:
nice catch!, removed 530cb59